File: app/src/main/java/com/example/tochkatest/view/gitUsers/FragmentGitUsersView.java
package com.example.tochkatest.view.gitUsers;

import com.example.tochkatest.model.git.GitUsers;
import com.example.tochkatest.model.git.InputData;

import java.util.ArrayList;

public interface FragmentGitUsersView {
    void recyclerUsers(ArrayList<GitUsers> arr);
    void recyclerInputUsers(ArrayList<GitUsers> arr);

}

File: app/src/main/java/com/example/tochkatest/view/gitUsers/FragmentGitUsersView.kt
package com.example.tochkatest.view.gitUsers

import com.example.tochkatest.model.git.GitUsers
import com.example.tochkatest.model.git.InputData

import java.util.ArrayList

interface FragmentGitUsersView {
    fun recyclerUsers(arr: ArrayList<GitUsers>)
    fun recyclerInputUsers(arr: ArrayList<GitUsers>)
    fun recyclerError()

}
